.travis.yml

```diff
@@ -10,7 +10,8 @@ before_install:
   - npm install -g grunt@0.4.5
   - npm install -g grunt-cli
 install: npm install
-scripts:
+script:
+  - grunt
```

Comment:
Yeah, I was using an incorrect version of `grunt-eslint`. I think I've got this sorted now though.